### PR TITLE
Init `GroupPermissionGrid` and `RolePermissionGrid`

### PR DESCRIPTION
**Pull request by dnlkoch**
This initializes the UI components for handling `Group-` and `Role` permissions for a given entity.

image

Requires [#698](https://github.com/terrestris/shogun-util/pull/698) and [#872](https://github.com/terrestris/shogun/pull/872).

### CODE REVIEW

File: src/Controller/GenericEntityController.ts [hunk 1]
```diff
@@ -163,16 +163,18 @@ export class GenericEntityController<T extends BaseEntity> {
...
       await this.service.update(entityUpdateObject as T) :
       await this.service.add(entityUpdateObject as T);
 
-    if (isPublic) {
-      await this.service.setPublic(this.entity.id!);
-    } else {
-      await this.service.revokePublic(this.entity.id!);
+    if (this.formConfig?.publicKey) {
+      const isPublic = this.entity?.[this.formConfig.publicKey as keyof T] as boolean;
+
```
Comment (KaiVolland) [inline src/Controller/GenericEntityController.ts:172]: Refactoring this to have the value of `this.formConfig?.publicKey` has a variable might enhance readability.

**Review by KaiVolland — APPROVED** [review]
Nice. Feel free to merge once the related dependencies have been released and updated.

File: src/Controller/GenericEntityController.ts (hunk 1 above)
Comment (dnlkoch) [reply src/Controller/GenericEntityController.ts]: Fixed in ebd3276be2d113c43782886a0d4ca665f8d575b4.

**Comment by dnlkoch** [comment]
Thanks for the review!

**Pull request merged**